blazegraph_load_test.sh

```diff
@@ -10,7 +10,69 @@ if [ -z "$BG_SPARQL" ]; then
 fi
 
 # ---
+# Create or recreate the test namespace
+
+# parse the namespace from the BG_SPARQL URI
+export bg_url=$(echo $BG_SPARQL | sed -e 's#\(.*\)/namespace.*#\1#')
+export bg_namespace=$(echo $BG_SPARQL | sed -e 's#.*namespace/\(.*\)/sparql#\1#')
```

Comment:
- It looks like you actually need to include the `/namespace` portion of the url here as you only use the `bg_url` while adding `/namespace`

- `sed` seems to be a heavyweight way to parse the url. Have you tried to use `cut` ? It would change these lines to:
```
bg_url=$(echo $BG_SPARQL | cut -d "/" -f 1,2,3,4)
bg_namespace=$(echo $BG_SPARQL | cut -d "/" -f 6)
```